compositor_render/src/transformations/image_renderer.rs

```diff
@@ -59,11 +61,22 @@ impl Image {
         Ok(renderer)
     }
 
-    fn download_file(url: &str) -> Result<bytes::Bytes, ImageError> {
-        // TODO: support local files
-        let response = reqwest::blocking::get(url)?;
-        let response = response.error_for_status()?;
-        Ok(response.bytes()?)
+    fn download_file(path_or_url: &str) -> Result<bytes::Bytes, ImageError> {
+        match Url::parse(path_or_url) {
+            Ok(_) => {
+                let response = reqwest::blocking::get(path_or_url)?;
+                let response = response.error_for_status()?;
+                Ok(response.bytes()?)
+            }
+            Err(err) => {
+                match fs::read(path_or_url) {
+                    Ok(file) => Ok(Bytes::from(file)),
+                    // Fallback to url parsing error if reading
+                    // from disk fails.
+                    Err(_) => Err(err.into()),
+                }
+            }
+        }
```

Comment:
I think it would be cleaner to handle that differently. Instead of this, we can accept the path provided by two options: "url" or "path"/"file"/"file_path"/etc. Passing the file path as "url" seems wrong to me, since it's usually reserved for web resources (https://en.wikipedia.org/wiki/URL).

Reply:
What about using `file://<path>`? Chromium uses file protocol for opening files. We could use the same approach to make it consistent with web renderer. Although, I'm not sure if web renderer/chromium should dictate how the rest of the compositor looks like

Reply:
- Passing the local path seems wrong to me, so I wanted to "hide" this feature a bit and only use that for our examples.
- file:// is not intuitive in my opinion especially when using it to define relative paths. I never remember how many slashes needs to be ther

I like the idea of having 2 fields path and url